tags

```diff
@@ -0,0 +1,1387 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
Remove this. If `tags` is not already part of `.gitignore`, add it. Also `*.patch` and `*.diff` so we don't check these in.

Reply:
Sorry man, I saw 1.patch in my diff. It was late night. :) I'll remove this